### ISSUE TITLE
notable does not start out of the box on debian unstable

### ISSUE THREAD

**Issue by jankatins** [notable/notable#792]
After installing notable_1.7.3_amd64.deb I got the following message:  

```
[17263:0823/202821.611989:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox 
helper binary was found, but is not configured correctly. Rather than run without
sandboxing I'm aborting now. You need to make sure that /opt/Notable/chrome-sandbox
is owned by root and has mode 4755.
```
and notable did not start (was wondering why clicking the icon didn't start the window...)

I did exactly that: `sudo chmod 4755 /opt/Notable/chrome-sandbox` and notable started afterwards.

- **OS Version**:
debian unstable, up to date from a few days ago

- **Notable Version**: 
1.7.3

**Comment by fabiospampinato** [comment]
This indeed seems to be an issue related to the sandbox, as @DallasO points out. I've actually removed the linked "fix" in recent versions because that caused other issues elsewhere.

I'll see how this can be addressed 🤔 

For future reference to myself:
- https://github.com/electron/electron/issues/16631#issuecomment-476082063
- https://github.com/wireapp/wire-desktop/pull/2966/files

I'll probably post a test `.deb` that you guys can try, even though maybe Debian  Testing might still not be supported because of another issue #725.

**Comment by yatesco** [comment]
Seeing the same thing here on Debian Stable.

**Comment by fabiospampinato** [comment]
This might have been fixed in v1.8, could somebody please check this?

**Comment by Baldomo** [comment]
I can confirm Notable 1.8.0 works OOTB on Linux (Manjaro 5.3.8)

![https://imgur.com/vEM1yeX.png](https://imgur.com/vEM1yeX.png)

**Comment by fabiospampinato** [comment]
@Baldomo is that debian-based or was it not working out of the box before?

**Comment by fabiospampinato** [comment]
It looks like this got fixed indeed https://github.com/notable/notable/issues/725. If somebody can still reproduce I'd be happy to reopen.

**Comment by Baldomo** [comment]
Manjaro is Arch-based, but the sandbox permission error was indipendent of the distro because electron would mess up on Linux in general

**Comment by DallasO** [comment]
I'm still seeing this with Debian Testing. Can someone else confirm?

Is there any logs or debugging I can help with?

**Comment by fabiospampinato** [comment]
> Is there any logs or debugging I can help with?

I don't think so, you could try starting the app from the terminal and see if anything is returned.

I'll try to reproduce this myself again. Did you try to install the `.deb` bundle or some other bundle?

**Comment by fabiospampinato** [comment]
If anybody knows how to fix this without breaking other things I'd be happy to take a closer look at it.

**Comment by DallasO** [comment]
> > Is there any logs or debugging I can help with?
> 
> I don't think so, you could try starting the app from the terminal and see if anything is returned.
> 
> I'll try to reproduce this myself again. Did you try to install the `.deb` bundle or some other bundle?

I'm using the `.deb` bundle.
Here is the error if ran from the command line:
```
[50541:0114/072310.780065:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /opt/Notable/chrome-sandbox is owned by root and has mode 4755.
Trace/breakpoint trap
```
And this appears in `syslog`:
```
Jan 14 07:23:10 Debian kernel: [ 3710.034471] traps: notable[50541] trap int3 ip:5628a54fce5f sp:7ffd60eafd70 error:0 in notable[5628a35f7000+53a5000]
```
Launching from Gnome Launcher has both the same logs in `syslog`.

**Comment by cormite** [comment]
I tried with both AppImage and .deb under debian 10.2 and I get in both cases

`The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that chrome-sandbox is owned by root and has mode 4755.`

**Comment by GogoAkira** [comment]
> It looks like this got fixed indeed #725. If somebody can still reproduce I'd be happy to reopen.

Still an issue Debian 'current' Buster.  Can't install deb file, will now try other ways, app looks decent :) need to have.

**Comment by fabiospampinato** [comment]
I think this might have been fixed in electron-builder v22 (https://github.com/electron-userland/electron-builder/pull/4163/files).

I'll see if I can upgrade to that sooner rather than later.

**Comment by username-is-not-available** [comment]
@fabiospampinato 
Same with Debian Stable too.

OS- Debian Stretch (stable)
Notable version- 1.8.4_amd64

Notable doesn't start by clicking on its icon. Running 'notable' from terminal returns following error:
`[15604:0413/172800.851603:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /opt/Notable/chrome-sandbox is owned by root and has mode 4755.
Trace/breakpoint trap`

@jankatins suggested: running `sudo chmod 4755 /opt/Notable/chrome-sandbox` notable started afterwards.

**Comment by gvisoc** [comment]
I'm experiencing this as well in Debian 10 Stable (exact same message as @elaboratebriefly), for both .deb and .appimage bundles. The `chmod 4755 /opt/Notable/chrome-sandbox` works for me as well.

**Comment by gioppoluca** [comment]
./Notable-1.8.4.AppImage 
[5701:0429/212321.758507:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /tmp/.mount_NotabloZS8ID/chrome-sandbox is owned by root and has mode 4755.
Trace/breakpoint trap (core dumped)

On CentOS 7
